crates/matrix-sdk/src/event_cache/pagination.rs

```diff
@@ -66,9 +66,9 @@ impl RoomPagination {
     /// that case, it's desirable to call the method again.
     // TODO(bnjbvr): that's bad API, this API should restart for us!
     #[instrument(skip(self))]
-    pub async fn paginate_backwards(&self, batch_size: u16) -> Result<BackPaginationOutcome> {
+    pub async fn run_backwards(&self, batch_size: u16) -> Result<BackPaginationOutcome> {
```

Comment:
```diff
- run_backwards
+ backwards
```

Thoughts?

Reply:
I like my effectful functions to start with verbs :eyes: Is that ok to keep it like that?

Reply:
I can live with that 😉